components/supabase/sources/new-webhook-event/new-webhook-event.mjs

```diff
@@ -0,0 +1,41 @@
+import base from "../common/base.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "supabase-new-webhook-event",
+  name: "New Webhook Event (Instant)",
+  description: "Emit new event for every `insert`, `update`, or `delete` operation in a table. This source requires user configuration using the Supabase website. More information in the README. [Also see documentation here](https://supabase.com/docs/guides/database/webhooks#creating-a-webhook)",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    ...base.props,
+    http: {
+      type: "$.interface.http",
+    },
+  },
+  hooks: {
+    async deploy() {
+      const client = await this.supabase._client();
+      const query = client
```

Comment:
```suggestion
      description: "The name of the table to watch for new rows.",
```

Reply:
Added a period at the end of the description to follow Google's Developer docs style guide.